### PR TITLE
Assignment no.2

### PR DESCRIPTION
**Pull request by vivekkumar677**

### CODE REVIEW

File: src/Assignment-2/assignment2.html [hunk 1]
```diff
@@ -8,21 +8,154 @@
...
-<body>
+<body style="background-color:lavender;">
 	<div id="container">
+		
+		<header style="border-style: solid; max-height: 100px; height: 100px; border-bottom-right-radius:15px; border-bottom-left-radius: 15px ; background-color:rgb(132, 150, 161);border-color: rgb(62, 118, 119);">
+			<div class="header-img">
+			  <img src="https://www.whatsappimages.in/wp-content/uploads/2020/06/Whatsapp-DP-Images-8-300x300.jpg">
+			</div>
+		
+			<div class="header-input">
```
Comment (manchalavaishnavi) [inline src/Assignment-2/assignment2.html]: instead of div tag use form tag for search input bar

File: src/Assignment-2/assignment2.html [hunk 2]
```diff
@@ -8,21 +8,154 @@
...
+					<a href="/Section 4/" style="color: rgb(15, 92, 117);">Section 4</a>
+				</h2>
+			</nav>
+		
+		  </header>	
+		  <div class="page-body">
+			<aside>
+			  <div>
+				<ul>
+				  <h3>
```
Comment (manchalavaishnavi) [inline src/Assignment-2/assignment2.html]: inside ul tag use li tag directly no need of h3 tag again. use css propertiles in div tag of aside for formatting

**Comment by manchalavaishnavi** [comment]
layout is very good

**Review by vivekkumar677 — COMMENTED** [review]
I changed my file of html & css

File: src/Assignment-2/assignment2.html [hunk 3]
```diff
@@ -8,21 +8,95 @@
 	<script src="bundle.js" type="text/javascript"></script>
 </head>
 
-<body>
+<body style="background-color:lavender;">
 	<div id="container">
-
+		
+		<header style="border-style: solid; max-height: 100px; height: 100px; border-bottom-right-radius:15px; border-bottom-left-radius: 15px ; background-color:rgb(132, 150, 161);border-color: rgb(62, 118, 119);">
+			<div class="header-img">
```
Comment (DyapaHarishwarReddy) [inline src/Assignment-2/assignment2.html:15]: Align the image in the header

File: src/Assignment-2/assignment2.html [hunk 4]
```diff
@@ -8,21 +8,95 @@
...
+			  <img src="https://www.whatsappimages.in/wp-content/uploads/2020/06/Whatsapp-DP-Images-8-300x300.jpg">
+			</div>
+		
+			<form class="header-input">
+				<input type="text" placeholder="Search" name="Search"/>
```
Comment (DyapaHarishwarReddy) [inline src/Assignment-2/assignment2.html:20]: Add search symbol

File: src/Assignment-2/assignment2.css [hunk 5]
```diff
@@ -1,7 +1,77 @@
...
+    margin: 10px;
+    margin-top: -10px;
+    max-height:35%;
+    max-width: 30%;
+}
+nav{
+    max-width: 40%;
+    margin: auto;
+}
+aside{
```
Comment (DyapaHarishwarReddy) [inline src/Assignment-2/assignment2.css:30]: Align the right sidebar just below the header

**Review by DyapaHarishwarReddy — COMMENTED** [review]
The layout is scrollable and right sidebar alignment to be corrected else everything is done proper.

**Pull request merged**